x-pack/plugins/observability_solution/entity_manager/server/templates/components/base_history.ts

```diff
@@ -7,8 +7,8 @@
 
 import { ClusterPutComponentTemplateRequest } from '@elastic/elasticsearch/lib/api/types';
 
-export const entitiesBaseComponentTemplateConfig: ClusterPutComponentTemplateRequest = {
-  name: 'entities_v1_base',
+export const entitiesHistoryBaseComponentTemplateConfig: ClusterPutComponentTemplateRequest = {
+  name: 'entities_v1_history_base',
```

Comment:
In https://github.com/elastic/kibana/pull/187507/commits/a54e47d68d98f7dee6502d9906ae1cf408a4967b I've added constants for these as well, so we'll need to update this code based on which PR gets merged first (I think I will wait for you to merge this)